website/releases/release-0.11.0.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Release 0.11.0"
+sidebar_position: 2
+layout: releases
+toc: true
+last_modified_at: 2022-01-27T22:07:00+08:00
+---
+# [Release 0.11.0](https://github.com/apache/hudi/releases/tag/release-0.11.0) ([docs](/docs/quick-start-guide))
+
+## Release Highlights
+
+### Multi-Modal Index
+
+In 0.11.0, we enable the [metadata table](/docs/metadata) with synchronous updates and
+metadata-table-based file listing by default with Spark engine, to improve the performance of partition and file listing
+on large Hudi tables. The metadata table and related file listing functionality can still be turned off by setting
```

Comment:
add 1 line saying "Due to this, users deploying Hudi with async table services need to configure a locking service. If this feature is not relevant for you, you can set `hoodie.metadata.enable=false` additionally and use Hudi as before."